build/maven.go

```diff
@@ -0,0 +1,264 @@
+package build
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	buildutil "github.com/jfrog/build-info-go/build/utils"
+	"github.com/jfrog/jfrog-client-go/utils/io/fileutils"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+)
+
+const (
+	MavenHome                       = "M2_HOME"
+	buildInfoPathKey                = "buildInfo.generated.build.info"
+	dependenciesDirName             = ".build-info"
+	MavenExtractorFileName          = "build-info-extractor-maven3-%s-uber.jar"
+	classworldsConfFileName         = "classworlds.conf"
+	PropertiesTempfolderName        = "properties"
+	mavenExtractorRemotePath        = "org/jfrog/buildinfo/build-info-extractor-maven3/%s"
+	GeneratedBuildInfoTempPrefix    = "generatedBuildInfo"
+	MavenExtractorDependencyVersion = "2.30.2"
```

Comment:
Let's update to the [latest](https://github.com/jfrog/build-info/blob/master/RELEASE.md) version. The same goes for gradle.